hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/trigger/Trigger.java

```diff
@@ -28,69 +34,144 @@
  * @author Jay Shaughnessy
  * @author Lucas Ponce
  */
-public class Trigger extends TriggerTemplate {
+public class Trigger {
```

Comment:
+1
Unifying Trigger and TriggerTemplate is good as it wasn't worked as a real template.
This makes the domain model more clean.
Thanks.